src/features/domain-preview/DomainPreview.tsx

```diff
@@ -0,0 +1,66 @@
+// React Imports
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+
+//- Utils Imports
+import { truncateAddress } from '../../lib/util/domains/domains';
+
+//- Constants Imports
+import { AddressTitle, ButtonTitle } from './DomainPreview.constants';
+
+type PreviewCardProps = {
+	title?: string;
+	description?: string;
+	icon?: string;
+	banner?: string;
+	href?: string;
+	owner?: string;
+	creator?: string;
+	isNFTView?: boolean;
+};
+
+const ViewContainerNFTCard: FC<PreviewCardProps> = ({
+	title,
+	description,
+	icon,
+	banner,
+	href,
+	owner,
+	creator,
+	isNFTView,
+}) => {
+	return (
+		<div style={{ margin: '16px 0' }}>
```

Comment:
Not sure if this is just a temp one, but we'll want to remove all inline styling.

Reply:
Done